### ISSUE TITLE
APIError: Cannot make Nylas request for account <account_id> auth token.

### ISSUE THREAD

**Issue by ryanpineo** [nylas/N1#2284]
##### Are there any related issues?

https://github.com/nylas/N1/issues/185 has the same error message but it doesn't seem related.

##### What operating system are you using?

Arch Linux kernel 4.5.4-1

##### What version of N1 are you using?

0.4.40 but it was also present in 0.4.33

**Bug?**
##### Do you have any third-party plugins installed?
no

##### Is the issue related to a specific email provider (Gmail, Exchange, etc.)?
I'm not sure.  I can only test with my gmail account.

##### Is the issue reproducible with a particular attachment, message, signature, etc?

Every time I add my gmail account the page is empty forever.  I see this message in the console:

```
ErrorReporter: Shipping Logs. Exception occurred
/usr/share/nylas/resources/app.asar/src/nylas-private-ship-logs-task
APIError: Cannot make Nylas request for account <account_id> auth token.
    at EventEmitter._electron.ipcMain.on (/usr/share/nylas/resources/app.asar/src/browser/application.js:591:17)
    at emitTwo (events.js:100:13)
    at EventEmitter.emit (events.js:185:7)
    at EventEmitter.<anonymous> (/usr/share/nylas/resources/electron.asar/browser/api/web-contents.js:141:25)
    at emitTwo (events.js:100:13)
    at EventEmitter.emit (events.js:185:7)
```

The nylas window looks like this:

http://imgur.com/eMBDVmv

Restarting Nylas doesn't change anything and I get the same error.  I've tried removing my .nylas directory and adding my account again but it's the same error.  Interestingly the account_id doesn't seem to change.  I also tried deauthing nylas from my gmail and adding it again but no luck.

**Comment by 2xG** [comment]
Same problem here

Nylas N1 v0.4.40-85cf726 installed with AUR
linux 4.5.4-1
Microsoft Exchange Office365 Outlook account

```
Streaming log data to /tmp/Nylas-N1-13729.log
App load time: 2107ms
ErrorReporter: Shipping Logs. Exception occurred
/usr/share/nylas/resources/app.asar/src/nylas-private-ship-logs-task
APIError: Cannot make Nylas request for account <id> auth token.
    at EventEmitter._electron.ipcMain.on (/usr/share/nylas/resources/app.asar/src/browser/application.js:591:17)
    at emitTwo (events.js:100:13)
    at EventEmitter.emit (events.js:185:7)
    at EventEmitter.<anonymous> (/usr/share/nylas/resources/electron.asar/browser/api/web-contents.js:141:25)
    at emitTwo (events.js:100:13)
    at EventEmitter.emit (events.js:185:7)
[SECURITY] node-uuid: crypto not usable, falling back to insecure Math.random()
Running log ship: /tmp, Nylas-N1-13729[.0-9]*.log$
APIError: Cannot make Nylas request for account <id> auth token.
    at EventEmitter._electron.ipcMain.on (/usr/share/nylas/resources/app.asar/src/browser/application.js:591:17)
    at emitTwo (events.js:100:13)
    at EventEmitter.emit (events.js:185:7)
    at EventEmitter.<anonymous> (/usr/share/nylas/resources/electron.asar/browser/api/web-contents.js:141:25)
    at emitTwo (events.js:100:13)
    at EventEmitter.emit (events.js:185:7)
Error uploading /tmp/Nylas-N1-13729.13776.log: status code 204
Error uploading /tmp/Nylas-N1-13729.log: status code 204
Error uploading /tmp/Nylas-N1-13729.13759.log: status code 204
ErrorReporter: Shipped Logs.
ErrorReporter: Shipping Logs. Exception occurred
/usr/share/nylas/resources/app.asar/src/nylas-private-ship-logs-task
[SECURITY] node-uuid: crypto not usable, falling back to insecure Math.random()
Running log ship: /tmp, Nylas-N1-13729[.0-9]*.log$
Error uploading /tmp/Nylas-N1-13729.13767.log: status code 204
ErrorReporter: Shipped Logs.
```

At the same time I have another computer with the almost the same software installed and all works good.

**Comment by chauncey** [comment]
I've had the same issue since sometime in late March around 0.4.19.

Arch Linux 4.5.4-1 x86_64
N1 0.4.45 from source currently but I've tried every release
Exchange and 2 gmail accounts
[12130:0614/202927:INFO:CONSOLE(306)] "APIError: Cannot make Nylas request for account `<id>` auth token.

This was working in both Arch and OS X for a few months. OS X still works great using the same auth token. I've removed every nylas/N1 directory and reinstalled several times on Arch. I have a support ticket in where they asked me to reinstall 0.4.40 after removing the .nylas directory but I haven't heard back yet. I've also tried adding an account on Arch that I didn't have on OS X thinking that might narrow it down. Everything seems to authenticate during setup, but I still have the APIError.
